sdk/eventgrid/azure-messaging-eventgrid-namespaces/src/main/java/com/azure/messaging/eventgrid/namespaces/EventGridServiceVersion.java

```diff
@@ -10,14 +10,6 @@
  */
 public enum EventGridServiceVersion implements ServiceVersion {
 
-    /**
-     * Enum value 2023-06-01-preview.
-     */
-    V2023_06_01_PREVIEW("2023-06-01-preview"),
-    /**
-     * Enum value 2023-10-01-preview.
-     */
-    V2023_10_01_PREVIEW("2023-10-01-preview"),
     /**
```

Comment:
0.15.14 changed the default behavior on ServiceVersion file.

Preview api-version will not be included, if the latest version is stable.
(data-plane preview api-version is supposed to only live for 3 months after their replacement)